File: 14. Longest Common Prefix/solution.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        out=""
        strs.sort()
        a=strs[0]
        b=strs[-1]
        for i in range(min(len(strs[0]),len(strs[-1]))):
            if a[i]==b[i]:
                out+=(a[i])
            else:
                break
        return out

File: 14. Longest Common Prefix/solution.java
class Solution {
    public String longestCommonPrefix(String[] strs) {
        String a,b,out="";
        int j;
        Arrays.sort(strs);
        a=strs[0];
        b=strs[strs.length-1];
        j=Math.min(a.length(),b.length());
        for(int i=0;i<j;i++){
            if (a.charAt(i)==b.charAt(i)){
                out+=(a.charAt(i));
            }
            else{
                break;
            }    
            
        }
        return out;
    }
}